deploy/libproject/deploy.rb

```diff
@@ -181,10 +191,18 @@ def deploy(cmd_name, args)
 
   op.parse.validate
 
+  if op.opts.account == "#{TEST_PROJECT}@appspot.gserviceaccount.com"
+    raise ArgumentError.new(
```

Comment:
Why is this account incompatible?

Reply:
Added a comment. tl;dr: fixable but probably not worth bothering